scripts/line-length.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+from typing import Optional
+from typing import Sequence
+from subprocess import call
+
+
+def main(argv: Optional[Sequence[str]] = None) -> int:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("filenames", nargs="*", help="Filenames to check")
+    args = parser.parse_args(argv)
+
+    for filename in args.filenames:
+        with open(filename) as fd:
+            for lineno, line in enumerate(fd):
+                if len(line) > 110:
+                    humanlineno = lineno + 1
+                    print(f"{filename}:{humanlineno} Line is too long.")
+                    lc = [
+                        "clang-format",
+                        "-i",
+                        f"-lines={humanlineno}:{humanlineno}",
+                        "-style={"
+                        "BasedOnStyle: Google, "
+                        "IndentWidth: 4, "
+                        "TabWidth: 8, "
+                        "UseTab: Never, "
+                        "ColumnLimit: 80, "
+                        # clang-format normally unbreaks all short lines,
+                        # which is undesired.
+                        # This does not happen here because line is too long
+                        "AccessModifierOffset: -2, "
+                        "AlignAfterOpenBracket: DontAlign, "
+                        "AlignOperands: false, "
+                        "AllowShortFunctionsOnASingleLine: None, "
+                        "AllowShortIfStatementsOnASingleLine: false, "
+                        "AllowShortLoopsOnASingleLine: false, "
+                        "BinPackArguments: false, "
+                        "BinPackParameters: false, "
+                        "ConstructorInitializerIndentWidth: 8, "
+                        "ContinuationIndentWidth: 8, "
+                        "IndentCaseLabels: false, "
+                        "DerivePointerAlignment: false, "
+                        "ReflowComments: false, "
+                        "SpaceAfterTemplateKeyword: false, "
+                        "SpacesBeforeTrailingComments: 1}",
+                        f"{filename}",
+                    ]
```

Comment:
Why are duplicating clang-format here? Is there no other way?

Reply:
I have tried hard, but that is the best I have found. 
The next better solution would be to parse the .clang-format and edit the ColumnLimit.
But that is IMHO overdone, because the .clang-format is no property of changes.

Reply:
What happens if you just specify the the line length on the command line? I'd expect that the config file is still loaded and the options specified on the command line override their config counterparts.

Reply:
That was my first attempt, but this fails. All kind of cwd tricks also not working.
Clang format always uses the .clang-format file next to the cpp file or the style Parameter. Not both.

Reply:
https://github.com/daschuer/mixxx/pull/53